### PR TITLE
JFB-2: ticket by stadium

### PR DESCRIPTION
**Pull request by RamazArtmeladze**
test comment for review

### CODE REVIEW

**Comment by JWstqT** [comment]
resolved conflict

**Comment by RamazArtmeladze** [comment]
conflict is resolved

File: out/production/gitHW3/TicketService.class
Comment (deolexx) [inline out/production/gitHW3/TicketService.class:1]: You should not store any class files at github

Comment (RamazArtmeladze) [reply out/production/gitHW3/TicketService.class:1]: I have deleted the .class files and .idea files. here is the PR link: https://github.com/RamazArtmeladze/AndersenProject/pull/4 
I also created a .gitignore file, which ensures that files like that will not uploaded to the repository, here is the PR link: 
https://github.com/RamazArtmeladze/AndersenProject/pull/5

**Pull request merged**